pkg/api/testing/defaulting_test.go

```diff
@@ -168,6 +168,10 @@ func TestDefaulting(t *testing.T) {
 		{Group: "scheduling.k8s.io", Version: "v1alpha1", Kind: "PriorityClassList"}:                            {},
 		{Group: "scheduling.k8s.io", Version: "v1beta1", Kind: "PriorityClassList"}:                             {},
 		{Group: "scheduling.k8s.io", Version: "v1", Kind: "PriorityClassList"}:                                  {},
+		{Group: "flowcontrol.apiserver.k8s.io", Version: "v1alpha1", Kind: "FlowSchema"}:                        {},
+		{Group: "flowcontrol.apiserver.k8s.io", Version: "v1alpha1", Kind: "FlowSchemaList"}:                    {},
+		{Group: "flowcontrol.apiserver.k8s.io", Version: "v1beta1", Kind: "FlowSchema"}:                         {},
+		{Group: "flowcontrol.apiserver.k8s.io", Version: "v1beta1", Kind: "FlowSchemaList"}:                     {},
```

Comment:
I believe this needs to be removed? I don't fully understand the fuzzer internals, but something similar is done here: https://github.com/kubernetes/kubernetes/pull/94244/files

Reply:
This is tripping CI: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/96527/pull-kubernetes-bazel-test/1327010193594126336

Reply:
Ah, yeah, if alpha wasn't already here then we probably shouldn't add either one. You might want to run that test locally to verify, since CI is slow.

Reply:
cc @lavalamp ^